src/main/scala/ru/smslv/akka/dns/raw/DnsClient.scala

```diff
@@ -61,6 +73,8 @@ class DnsClient(ns: InetSocketAddress, upstream: ActorRef) extends Actor with Ac
   }
 }
 
+case class SrvQuestion(id: Short, name: String)
```

Comment:
Would you consider renaming to `SRVQuestion` to match other SRV related class names?